src/App.css

```diff
@@ -5,16 +5,25 @@
 
 body {
   padding: 24px;
+  background-color: #f5f6fa;
 }
 
 main {
```

Comment:
- [ ] For CSS best practices kindly use class to style elements instead of naming tags